lib/pbench/test/unit/server/test_requests.py

```diff
@@ -220,10 +259,10 @@ def test_upload(client, pytestconfig, caplog, server_config):
             assert response.status_code == 201, repr(response)
             sfilename = secure_filename(filename)
             tmp_d = pytestconfig.cache.get("TMP", None)
-            receive_dir = os.path.join(
+            receive_dir = Path(
                 tmp_d, "srv", "pbench", "pbench-move-results-receive", "fs-version-002"
             )
-            assert os.path.exists(receive_dir), (
+            assert (
```

Comment:
?? What are you actually trying to assert here???

Reply:
I removed the check `os.path.exists` here as I am using python `Path` to create `receive_dir` and it already offers an existence check.